tests/python/pants_test/engine/util.py

```diff
@@ -145,9 +147,10 @@ def remove_locations_from_traceback(trace):
 class MockConsole(object):
   """An implementation of pants.engine.console.Console which captures output."""
 
-  def __init__(self):
+  def __init__(self, use_colors=True):
```

Comment:
Maybe for a followup: defaulting this to false (or using false explicitly in tests) would avoid needing to strip things in most cases.

Reply:
Ack. Realized it was important parametrize this and to not hardcode for exactly that reason.

I'm thinking that we'll want callers of this to explicitly say that they don't want colors—to opt out, rather than opt in.

Usually for unit tests, we will want to check that colors are being rendered properly (iirc we do this for V1 in some places), so I think it's good to default to this here.